docs/dev/zksync-v2/l1-l2-interop.md

```diff
@@ -2,68 +2,72 @@
 
 In other words, we require the operator to do its best instead of requiring a strict deadline. The measure of "the work" is still to be developed. Most likely it will be the number of `ergs` the priority operations used.
 
-Fourthly, there needs to be a way to bypass spam attacks for important transactions. From the points above we already know that there is no strict deadline for processing the transactions. If the deque with priority gets filled with a lot of heavy transactions, the next transactions will need to wait until all these are processed. If you need to pass your transaction urgently, you can submit your transaction to a separate priority heap, where the transactions will be processed based on the layer 2 tip (`tx.value - txBaseCost * gasPrice`) provided by the operator. The transactions in the heap get processed before transactions in the queue.
+Fourthly, there needs to be a way to bypass spam attacks for important transactions. From the points above we already know that there is no strict deadline for processing the priority queue transactions. If the deque with priority gets filled with a lot of heavy transactions, the next transactions will need to wait until all these are processed. If you need to pass your transaction urgently, you can submit your transaction to a separate priority heap, where the transactions will be processed based on the layer 2 tip (`tx.value - txBaseCost * gasPrice`) provided to the operator. The transactions in the heap get processed before transactions in the queue.
 
 ### Summary
 
-For each transaction, besides supplying the transaction input data, the user also provides the following:
+For each priority queue transaction, besides supplying the transaction input data, the user also provides the following:
 
 - The queue type to use. It is either a `Deque`, `HeapBuffer` or `Heap`. `Deque` is used for sequential processing of the transactions, while `HeapBuffer` and `Heap` are used for fee-prioritized processing of transactions. For the testnet, only `Deque` is available. The details on using `HeapBuffer` and `Heap` will be available later with their release.
 - The operator fee for the transaction as part of the transaction `value` (the amount of ETH passed with the L1 transaction).
 
 ## Priority mode
 
-If the operator fails to process the needed L1 transactions, the system enters the Priority mode. In this mode, everyone can become an operator by staking tokens. The exact details of the priority mode are still under development and will be described in more detail closer to the mainnet launch.
+If the operator fails to process the needed L1 transactions, the system enters the ''Priority mode'''. In this mode, everyone can become an operator by staking tokens. The exact details of the priority mode are still under development and will be described in more detail closer to the mainnet launch.
```

Comment:
Two opening `''` and three closing `'''`

Reply:
what??

Reply:
> ''Priority mode'''

Reply:
Fixed!